Tests/GRDBTests/FailureTestCase.swift

```diff
@@ -1,6 +1,7 @@
 // Inspired by https://github.com/groue/CombineExpectations
 import XCTest
 
+#if canImport(Darwin) // needed for XCTIssue
```

Comment:
`FailureTestCase` is the base class of `ValueObservationRecorderTests`, which tests `ValueObservationRecorder`. `ValueObservationRecorder`, on its turn, is used to test `ValueObservation`. And `ValueObservation` is a GRDB piece that must remain super reliable (i.e. tested very aggressively).

We'll need to find a way to restore `ValueObservationRecorder` tests.

For the time being, let's restore this class, and have `assertFailure` throw `XTCSkip` in environments that don't have XCTIssue (replacing `rethrows` with `throws`).

Reply:
Done in [15e3089](https://github.com/groue/GRDB.swift/pull/1708/commits/15e308950c1f47adcedf4ad80a303b95e022c0f9)